packages/jest-expo/src/preset/expoModules.js

```diff
@@ -388,7 +391,7 @@ module.exports = {
         ExpoRandom: [{ key: 0, argumentsCount: 1, name: 'getRandomBase64StringAsync' }],
         ExpoSMS: [
           { key: 0, argumentsCount: 0, name: 'isAvailableAsync' },
-          { key: 1, argumentsCount: 2, name: 'sendSMSAsync' },
+          { key: 1, argumentsCount: 3, name: 'sendSMSAsync' },
```

Comment:
I think these were recently updated on `master`, would you care to rebase on top of it?